manipvortex/manipulator.go

```diff
@@ -594,9 +594,13 @@ func (m *vortexManipulator) backgroundSync(ctx context.Context) {
 		return
 	}
 
+	limiter := time.Tick(200 * time.Millisecond)
```

Comment:
can you use a https://godoc.org/golang.org/x/time/rate ?

Reply:
also the 200ms should be configurable by an option

Reply:
Fixed.